packages/apps/tasks-app/src/App.tsx

```diff
@@ -48,7 +49,7 @@ export const App = () => {
   return (
     <ThemeProvider
       appNs='halo'
-      resourceExtensions={[appkitTranslations, tasksTranslations]}
+      resourceExtensions={[appkitTranslations, osTranslations, tasksTranslations]}
```

Comment:
is order important? if there's a collision which takes precedence?

Reply:
Order isn't important unless the resources define translations under the same namespace, in which case i18n will overwrite any keys within that namespace that collide. We’ve been avoiding that as a practice though, so order doesn’t currently matter.